riotfile.py

```diff
@@ -2504,18 +2504,26 @@ def select_pys(min_version: str = MIN_PYTHON_VERSION, max_version: str = MAX_PYT
                     },
                 ),
                 Venv(
-                    pys=select_pys(min_version="3.7", max_version="3.11"),
+                    pys="3.7",
```

Comment:
was it intentional to stop testing >=3.7 here?

Reply:
It looks like we are, but this is more of a testing shuffle (and dropping tests for v1.1.1 which are unnecessary imo) rather than stopping testing for >=3.7. We went from testing:

- Python 3.7 with v0.26
- Python 3.7 - 3.11 with v1.1.1 and v1.30.1
- Python 3.8 - 3.11 with v1.30.1

To:

- Python 3.7 with v0.26 and v1.30.1
- Python 3.8-3.11 with v1.30.1
- Python 3.8-3.13 with v1.60.0 (latest)